CMakeLists.txt

```diff
@@ -53,6 +53,12 @@ install(DIRECTORY ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/ DESTINATION ${CMAKE_INSTALL
 install(DIRECTORY pd-help/ DESTINATION ${CMAKE_INSTALL_PREFIX})
 
 #Dependencies: availability
+#Non-standard openssl path on OSX
+if(${CMAKE_SYSTEM_NAME} STREQUAL "Darwin")
```

Comment:
I know, it is often the default location.
But hardcoding library locations is really ugly, as it depends on the user of the system. where he wants his libraries.
Therefore, I would rather avoid hardcoding the location and instead presenting a useful info to a user,

What do you think about:

find_package(OpenSSL)
if (NOT OPENSSL_FOUND)
  message(info, "OpenSSL could not be found. Please install or, if it is installed, provide its location by setting OPENSSL_ROOT, e.g., cmake -DOPENSSL_ROOT_DIR=/usr/local/ssl")
endif()